app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.exceptions.ProductNotFoundException;
+import javax.transaction.Transactional;
+import org.springframework.stereotype.Service;
+
+/**
+ * 상품 목록 관리를 담당합니다.
+ */
+@Service
+@Transactional
+public class ProductService {
+
+    ProductRepository productRepository;
```

Comment:
이렇게 해 주는 것이 좀 더 좋겠죠.
```suggestion
    private final ProductRepository productRepository;
```

Reply:
앗 이부분을 빼먹었군요..